[CS] Code Style fixes for unit tests

photodude opened the pull request:
Pull Request for Issue code style issues.

### Summary of Changes
- Object operator not indented correctly
- Tabs must be used to indent lines; spaces are not allowed

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style testing on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to implement fixable code style

### Documentation Changes Required
none

photodude committed:
PHPCS2 Autofixers

- Object operator not indented correctly
- Tabs must be used to indent lines; spaces are not allowed

photodude committed:
PHPCS2- autofixers

reapply autofixers to fix space indents

Quy commented:
I have tested this item :white_check_mark: successfully on 0982e9d97bf4746e53d78e7918a64ae4196279ba<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19155">issues.joomla.org/tracker/joomla-cms/19155</a>.</sub>

wilsonge commented on tests/unit/suites/database/driver/mysql/JDatabaseImporterMysqlTest.php:
```diff
@@ -61,7 +61,7 @@ public static function setUpBeforeClass()
 		$this->dbo = $this->getMockBuilder('JDatabaseDriverMysql')
-					->setMethods(array(
+			->setMethods(array(
 						'getErrorNum',
```
Can we fix array elements spacing at the same time? looks a bit crazy now :)

And this goes for all the JDatabaseImporter and JDatabaseExporter methods touched this way

photodude replied:
I'll rebase and run the full code style on these files.

photodude replied:
I've done some tweaking of the array elements on a couple of the files. It doesn't seem like the full code style rules are adjusting anything with these array elements.

wilsonge commented on tests/unit/suites/database/driver/mysqli/JDatabaseImporterMysqliTest.php:
```diff
@@ -47,7 +47,7 @@ class JDatabaseImporterMysqliTest extends \PHPUnit\Framework\TestCase
 		$this->dbo = $this->getMockBuilder('JDatabaseDriverMysqli')
-					->setMethods(array(
+			->setMethods(array(
 						'getErrorNum',
```
and here

photodude committed:
autofix code style errors

- End of line character is invalid; expected "\n" but found "\r\n"
- Multi-line function call not indented correctly
- Closing parenthesis of a multi-line function call must be on a line by
itself

photodude committed:
more autofixers

- End of line character is invalid; expected "\n" but found "\r\n"
- Multi-line function call not indented correctly
- Closing parenthesis of a multi-line function call must be on a line by
itself
- Single line block comment not allowed; use inline ("// text") comment
instead
- Block comment text must start on a new line
- Expected 1 space(s) before asterisk; 0 found
- Comment must start with a capital letter; found "f"

Quy commented on tests/unit/suites/database/driver/mysql/JDatabaseDriverMysqlTest.php:
```diff
@@ -542,7 +542,8 @@ public function testLoadNextRow($rowArr)
 
-		/* last call to free cursor, asserting that returns false */
+		// Last call to free cursor, asserting that returns false
+
```
Delete blank line.

photodude replied:
ugh, I thought I had caught all of these. Should be fixed now

Quy commented on tests/unit/suites/database/driver/mysql/JDatabaseDriverMysqlTest.php:
```diff
@@ -843,13 +844,14 @@ public function testTransactionRollback($toSavepoint, $tupleCount)
 
-		/* find how many rows have description='testRollbackSp' :
+		/*
+		 find how many rows have description='testRollbackSp' :
```
Add *

Quy commented on tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php:
```diff
@@ -469,7 +469,8 @@ public function testLoadNextObject($objArr)
 
-		/* last call to free cursor, asserting that returns false */
+		// Last call to free cursor, asserting that returns false
+
```
Delete blank line.

Quy commented on tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php:
```diff
@@ -602,7 +603,8 @@ public function testLoadNextRow_plusLoad($rowArr)
 
-		/* last call to free cursor, asserting that returns false */
+		// Last call to free cursor, asserting that returns false
+
```
Delete blank line.

Quy commented on tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php:
```diff
@@ -877,20 +879,21 @@ public function testTransactionRollback($toSavepoint, $tupleCount)
 
-		/* create savepoint only if is passed by data provider */
+		// Create savepoint only if is passed by data provider
+
```
Delete blank line.

Quy commented on tests/unit/suites/database/driver/mysqli/JDatabaseDriverMysqliTest.php:
```diff
@@ -899,13 +902,14 @@ public function testTransactionRollback($toSavepoint, $tupleCount)
 
-		/* find how many rows have description='testRollbackSp' :
+		/*
+		 find how many rows have description='testRollbackSp' :
```
Add *

Quy commented on tests/unit/suites/database/driver/pdomysql/JDatabaseDriverPdomysqlTest.php:
```diff
@@ -280,7 +280,8 @@ public function testGetTableColumns()
 
-		/* Not only type field */
+		// Not only type field
+
```
Delete blank line?

Quy commented on tests/unit/suites/database/driver/pdomysql/JDatabaseDriverPdomysqlTest.php:
```diff
@@ -793,13 +795,14 @@ public function testTransactionRollback($toSavepoint, $tupleCount)
 
-		/* Find how many rows have description='testRollbackSp' :
+		/*
+		 Find how many rows have description='testRollbackSp' :
```
Add *

Quy commented on tests/unit/suites/database/driver/postgresql/JDatabaseDriverPostgresqlTest.php:
```diff
@@ -1134,13 +1150,14 @@ public function testTransactionRollback($toSavepoint, $tupleCount)
 
-		/* find how many rows have description='testRollbackSp' :
+		/*
+		 find how many rows have description='testRollbackSp' :
```
Add *

photodude committed:
correct minor items

- remove newline after comments were fixed
- missing asterisk

photodude committed:
minor corrections

- Delete blank line after comments
- add asterisk to block comment

photodude committed:
minor corrections

- delete blank line after comment
- fix block comment

photodude committed:
minor corrections

- fix comment block style

photodude committed:
Comments start with capital letter

photodude committed:
adjust array alignments

photodude committed:
Array alignment

photodude committed:
array alignment

Quy commented on tests/unit/suites/database/driver/postgresql/JDatabaseDriverPostgresqlTest.php:
```diff
@@ -1222,23 +1239,24 @@ public function testRenameTable()
+		// Check index change
 		self::$driver->setQuery(
 			'SELECT relname
 							FROM pg_class
```
Reduce indentation

Quy commented on tests/unit/suites/database/driver/postgresql/JDatabaseDriverPostgresqlTest.php:
```diff
@@ -1222,23 +1239,24 @@ public function testRenameTable()
+		// Check sequence change
 		self::$driver->setQuery(
 			'SELECT relname
 							FROM pg_class
```
Reduce indentation

Quy commented on tests/unit/suites/database/driver/postgresql/JDatabaseImporterPostgresqlTest.php:
```diff
@@ -32,7 +32,7 @@ public function setup()
 		$this->dbo = $this->getMockBuilder('JDatabaseDriverPostgresql')
-					->setMethods(array(
+			->setMethods(array(
 						'getErrorNum',
```
Reduce indentation
